### PR TITLE
timestamp/time: have the option of sending an initial packet.

### PR DESCRIPTION
**Pull request by glima**
Signed-off-by: Gustavo Lima Chaves <gustavo.lima.chaves@intel.com>

### CODE REVIEW

File: src/modules/flow/timestamp/timestamp.json [hunk 1]
```diff
@@ -21,7 +21,27 @@
...
+            "data_type": "boolean",
+            "default": true,
+            "description": "Whether to create an initial packet and send it upon connection to the port OUT or not (wait for packets in the TRIGGER port).",
+            "name": "initial_packet"
+          },
+          {
+            "data_type": "boolean",
+            "default": false,
+            "description": "Ignore case when comparing strings.",
+            "name": "ignore_case"
```
Comment (edersondisouza) [inline src/modules/flow/timestamp/timestamp.json]: What is this option about? It seems to not have any code associated to it...

Comment (glima) [inline src/modules/flow/timestamp/timestamp.json]: The famous wat. I'll remove that extra option coming from c&p, thanks.

**Comment by bdilly** [comment]
Patch seems good removing "ignore_case" option.

**Comment by glima** [comment]
Alright, fixing locally and merging. Thanks.

**Pull request closed without merging**